tools/pybind11Tools.cmake

```diff
@@ -12,10 +12,20 @@ if(pybind11_FIND_QUIETLY)
   set(_pybind11_quiet QUIET)
 endif()
 
-# Add a CMake parameter for choosing a desired Python version
-if(NOT PYBIND11_PYTHON_VERSION)
+# If this is the first run, PYTHON_VERSION can stand in for PYBIND11_PYTHON_VERSION
+if(NOT DEFINED PYBIND11_PYTHON_VERSION AND DEFINED PYTHON_VERSION)
```

Comment:
So is this part only executed in "classic mode" (as the PR title suggests)?

I'm wondering if having set `PYBIND11_PYTHON_VERSION` or `PYTHON_VERSION` ought to give an error/warning when `FindPython` is used? (since it won't have any effect, right?)

Reply:
It will give an "unused variable specified" warning in CMake already.

Reply:
Alright!